<ui version="4.0" >
 <class>KCometen4UI</class>
 <widget class="QWidget" name="KCometen4UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Screen Saver Config</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" >
   <item>
    <widget class="QTabWidget" name="tabWidget3" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Scene</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10" >
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Camera</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <item>
             <widget class="QLabel" name="textLabelFov" >
              <property name="whatsThis" >
               <string>Set the field of view for the widest viewport dimension. Values range from 10 to 120 degrees.</string>
              </property>
              <property name="text" >
               <string>Field of view:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="kDoubleNumInputFov" >
              <property name="whatsThis" >
               <string>Set the field of view for the widest viewport dimension. Values range from 10 to 120 degrees.</string>
              </property>
              <property name="value" >
               <double>76.000000000000000</double>
              </property>
              <property name="minimum" >
               <double>10.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>120.000000000000000</double>
              </property>
              <property name="suffix" >
               <string>Â°</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkFreeCamera" >
            <property name="whatsThis" >
             <string>Toggle between a free floating or fixed position camera.</string>
            </property>
            <property name="text" >
             <string>Free camera</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>23</width>
                  <height>29</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="textLabel1_2" >
                <property name="whatsThis" >
                 <string>Set the camera speed. Values range from 0 to 10.0.</string>
                </property>
                <property name="text" >
                 <string>Camera speed:</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="kDoubleNumInputCamSpeed" >
              <property name="whatsThis" >
               <string>Set the camera speed. Values range from 0 to 10.0.</string>
              </property>
              <property name="value" >
               <double>1.500000000000000</double>
              </property>
              <property name="maximum" >
               <double>10.000000000000000</double>
              </property>
              <property name="suffix" >
               <string> x</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Camera Effects</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9" >
          <item>
           <widget class="QCheckBox" name="checkSlowmotion" >
            <property name="whatsThis" >
             <string>With this option set, a slow motion camera will occasionally watch the action from a corner of the box.</string>
            </property>
            <property name="text" >
             <string>Slow motion with fixed camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkRotCam" >
            <property name="whatsThis" >
             <string>With this option set, a slow motion camera will occasionally rotate around the center of action.</string>
            </property>
            <property name="text" >
             <string>Slow motion with rotating camera</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" >
            <item>
             <widget class="QLabel" name="aspectLabel" >
              <property name="whatsThis" >
               <string>Adjust the apparent aspect ratio to make the slow-motion effect more "theatrical"</string>
              </property>
              <property name="text" >
               <string>Effect aspect ratio:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxAspectRatio" >
              <property name="whatsThis" >
               <string>Adjust the apparent aspect ratio to make the slow-motion effect more "theatrical"</string>
              </property>
              <item>
               <property name="text" >
                <string>1.85:1</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>2.40:1</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Full Screen</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="QLabel" name="textLabel3_2" >
           <property name="whatsThis" >
            <string>Adjust the size of the box relative to the comet size. Values range from 0.5 to 5.0.</string>
           </property>
           <property name="text" >
            <string>Size scale:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KDoubleNumInput" name="kDoubleNumInputSizeScale" >
           <property name="whatsThis" >
            <string>Adjust the size of box relative to the comet size. Values range from 0.5 to 5.0.</string>
           </property>
           <property name="value" >
            <double>2.500000000000000</double>
           </property>
           <property name="minimum" >
            <double>0.500000000000000</double>
           </property>
           <property name="maximum" >
            <double>5.000000000000000</double>
           </property>
           <property name="suffix" >
            <string> x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QLabel" name="textLabel3" >
           <property name="whatsThis" >
            <string>Set the maximum frames per second. Values range from 10 to 999.</string>
           </property>
           <property name="text" >
            <string>Max FPS:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntNumInput" name="kIntNumInputFps" >
           <property name="whatsThis" >
            <string>Set the maximum frames per second. Values range from 10 to 999.</string>
           </property>
           <property name="value" >
            <number>60</number>
           </property>
           <property name="minimum" >
            <number>10</number>
           </property>
           <property name="maximum" >
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkIsCube" >
         <property name="whatsThis" >
          <string>Set whether the internal scene should always be a cube or whether the screensaver should adjust it match the parent window proportions.</string>
         </property>
         <property name="text" >
          <string>Scene is always a cube</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMipmaps" >
         <property name="whatsThis" >
          <string>Checking this will possibly improve the image quality of the particles and background image.</string>
         </property>
         <property name="text" >
          <string>Use mipmaps</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Comets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" >
           <item>
            <widget class="QLabel" name="textLabel1" >
             <property name="whatsThis" >
              <string>Select the color of the comets.</string>
             </property>
             <property name="text" >
              <string>Comet color:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel2" >
             <property name="whatsThis" >
              <string>Set the approximate number of seconds that a comet will be created. Values range from 0.1 to 10.0 seconds.</string>
             </property>
             <property name="text" >
              <string>Comet interval:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textLabel4" >
             <property name="whatsThis" >
              <string>Adjust the comet speed relative to normal time. Values range from 0.1 to 5.0.</string>
             </property>
             <property name="text" >
              <string>Comet speed:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="densityLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="whatsThis" >
              <string>Adjust the density of the particles in the comet tails.</string>
             </property>
             <property name="text" >
              <string>Particle density:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7" >
           <item>
            <widget class="QComboBox" name="comboBoxColor" >
             <property name="whatsThis" >
              <string>Select the color of the comets.</string>
             </property>
             <item>
              <property name="text" >
               <string>Red</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Blue</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Green</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>All Colors</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Random Colors</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="KDoubleNumInput" name="kDoubleNumInputInterval" >
             <property name="whatsThis" >
              <string>Set the approximate number of seconds that a comet will be created. Values range from 0.1 to 10.0 seconds.</string>
             </property>
             <property name="value" >
              <double>1.250000000000000</double>
             </property>
             <property name="minimum" >
              <double>0.100000000000000</double>
             </property>
             <property name="maximum" >
              <double>10.000000000000000</double>
             </property>
             <property name="suffix" >
              <string> sec</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KDoubleNumInput" name="kDoubleNumInputTimeScale" >
             <property name="whatsThis" >
              <string>Adjust the comet speed relative to normal time. Values range from 0.1 to 5.0.</string>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
             <property name="minimum" >
              <double>0.100000000000000</double>
             </property>
             <property name="maximum" >
              <double>5.000000000000000</double>
             </property>
             <property name="suffix" >
              <string> x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KIntNumInput" name="kIntNumInputDensity" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="whatsThis" >
              <string>Adjust the density of the particles in the comet tails.</string>
             </property>
             <property name="value" >
              <number>100</number>
             </property>
             <property name="minimum" >
              <number>0</number>
             </property>
             <property name="maximum" >
              <number>100</number>
             </property>
             <property name="suffix" >
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Comet Types</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <widget class="QCheckBox" name="checkRotateComet" >
            <property name="whatsThis" >
             <string>Create comets that have two heads which orbit each other.</string>
            </property>
            <property name="text" >
             <string>Binary comets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkSplitComet" >
            <property name="whatsThis" >
             <string>Create comets that end in a cascade of mini-explosions.</string>
            </property>
            <property name="text" >
             <string>Cascading explosions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkCurveComet" >
            <property name="whatsThis" >
             <string>Create comets that follow random curved paths.</string>
            </property>
            <property name="text" >
             <string>Curved comets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBlitz" >
            <property name="whatsThis" >
             <string>Create comet-destroying lightning.</string>
            </property>
            <property name="text" >
             <string>Lightning</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>checkBlitz</zorder>
         <zorder>checkCurveComet</zorder>
         <zorder>checkRotateComet</zorder>
         <zorder>checkSplitComet</zorder>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>Background</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Image Source</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QRadioButton" name="radioButton3" >
            <property name="whatsThis" >
             <string>Use the desktop as the box face image.</string>
            </property>
            <property name="text" >
             <string>Desktop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton4" >
            <property name="whatsThis" >
             <string>Use the selected file as the box face image.</string>
            </property>
            <property name="text" >
             <string>File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KUrlRequester" name="kURLRequesterFile" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="whatsThis" >
             <string>Use the selected file as the box face image.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton5" >
            <property name="whatsThis" >
             <string>Use random images from this directory as the box face image.</string>
            </property>
            <property name="text" >
             <string>Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KUrlRequester" name="kURLRequesterRandomDirectory" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="whatsThis" >
             <string>Use random images from this directory as the box face image.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="textLabel2_2" >
           <property name="whatsThis" >
            <string>Set the size that the background image will be scaled to.  This will change the image's aspect ratio, if necessary.</string>
           </property>
           <property name="text" >
            <string>Scaled size:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxBgSize" >
           <property name="whatsThis" >
            <string>Set the size that the background image will be scaled to.  This will change the image's aspect ratio, if necessary.</string>
           </property>
           <item>
            <property name="text" >
             <string>128x128</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>256x256</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>512x512</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>1024x1024</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Real Size</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="preview" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton5</sender>
   <signal>toggled(bool)</signal>
   <receiver>kURLRequesterRandomDirectory</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>69</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFreeCamera</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>263</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFreeCamera</sender>
   <signal>toggled(bool)</signal>
   <receiver>kDoubleNumInputCamSpeed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>210</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel" >
     <x>188</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton4</sender>
   <signal>toggled(bool)</signal>
   <receiver>kURLRequesterFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>77</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>102</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
